index.js

```diff
@@ -105,10 +105,13 @@ function Xray () {
           var $ = load(html, url)
           node.html($, next)
         })
-      } else {
+      } else if (source) {
         // `url` is probably HTML
```

Comment:
It's not my comment, but yeah, at this point the `source` is almost certainly HTML. I would remove it or possibly reword it. Because `source` is set to null in the `params` function, it is not entirely obvious here why `source` is null, so there might be some value in a comment.

I will leave it to you to decide, let me know and I will squash a change into my commit